modules/ding_tabroll/js/ding_tabroll.js

```diff
@@ -4,50 +4,53 @@
  * events on the tabs.
  */
 (function ($) {
-
-  $(document).ready(function($) {
-    var tabroll = $('.ding-tabroll');
-    var tabroll_select = $('.ding-tabroll-select-tabs');
-
-    // Hack to check if tab have been tab_selected, as unbind event will not work.
-    var tab_selected = false;
-
-    // Check if the tabs lib is loaded before trying to call it.
-    if ($.fn.tabs) {
-      tabroll.tabs({
-        select: function(event, ui) {
-          // Update the mobile navigation drop down.
-          tabroll_select.prop('selectedIndex', ui.index);
-        }
-      }).tabs("rotate", 5000);
-
-      // Stop tabs rotate when mouse is over the tab roll.
-      tabroll.mouseenter(function() {
-        tabroll.tabs('rotate', 0);
+  'use strict';
+
+  Drupal.behaviors.ding_tabroll = {
+    attach: function (context) {
+      var tabroll = $('.ding-tabroll', context);
+      var tabroll_select = $('.ding-tabroll-select-tabs', context);
+
+      // Hack to check if tab have been tab_selected, as unbind event will not work.
+      var tab_selected = false;
+
+      // Check if the tabs lib is loaded before trying to call it.
+      if ($.fn.tabs) {
+        tabroll.tabs({
+          select: function(event, ui) {
+            // Update the mobile navigation drop down.
+            tabroll_select.prop('selectedIndex', ui.index);
+          }
+        }).tabs("rotate", 5000);
```

Comment:
Nitpicking but this also uses double quotes?

Reply:
Fixed it :-)